src/EventListener/CronjobListener.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace Terminal42\LeadsBundle\EventListener;
+
+use Contao\CoreBundle\Monolog\ContaoContext;
+use Psr\Log\LoggerInterface;
+use Psr\Log\LogLevel;
+use Symfony\Bundle\FrameworkBundle\Console\Application;
+use Symfony\Component\Console\Input\ArrayInput;
+use Symfony\Component\Console\Output\BufferedOutput;
+use Symfony\Component\HttpKernel\KernelInterface;
+
+class CronjobListener
+{
+
+    /**
+     * @var KernelInterface
+     */
+    private $kernel;
+
+    /**
+     * @var LoggerInterface
+     */
+    private $logger;
+
+    /**
+     * CronjobListener constructor.
+     * @param KernelInterface $kernel
+     * @param LoggerInterface $logger
+     */
+    public function __construct(KernelInterface $kernel, LoggerInterface $logger)
+    {
+        $this->kernel = $kernel;
+        $this->logger = $logger;
+    }
+
+    function onDaily()
+    {
+
+        try {
+            $application = new Application($this->kernel);
```

Comment:
This approach seems odd to me. One should not create a command line application in a cron job to clear files. The correct approach would be to use a service that does the purging, which is then triggered from the command and from the cronjob listener.

Reply:
well, kind of historically grown :-/
a service would be the better way!